### DISCUSSION TITLE
URGENT 🚨 Talk proposal at ArgoCon about  "CDaaS: As-Service Offering with ArgoCD"

### DISCUSSION THREAD

**Discussion by abdennour** [argoproj/argo-cd#7989] [Show and tell]
Hello 👋 Guys !

I am a leading DevOps team at a big organization. We introduced ArgoCD since 1 year for rolling out DevOps transformation.

While introducing it, we focused on :
- How to make the fast rollout for more than 200 projects ?
- How to make it self-service ? 
- How to kill/reduce administrative tasks on ArgoCD ?
- How to reduce the learning curve for 500+ engineers ( between Dev/QA/Ops) while adding new component (ArgoCD) to the DevOps platform ?
- And in general, how to make it CD as a Service or GitOps as a Service ? 

We, really, want to share how we built that on top of ArgoCD. 
We just want to share it.

How can we proceed ?
 How can we contribute for ArgoCon with a TALK.

> Hints : 

> I opened a [lot of issues](https://github.com/argoproj/argo-cd/issues?q=is%3Aissue+author%3Aabdennour+) 😁 since we started the implementation. Check them please to get more insight about our activities [here](https://github.com/argoproj/argo-cd/issues?q=is%3Aissue+author%3Aabdennour+).

> I am contributor in the helm chart repo .i.e. https://github.com/argoproj/argo-helm/pull/603
Thank you

**Comment by jannfis** [comment]
Hi @abdennour, thanks for your offer to speak about your journey with Argo CD. 

ArgoCon '21 has been held at Dec 8th 2021 and is over by now. We're definitely planning for an ArgoCon '22, which will have a Call for Papers at a yet unknown time. I'd suggest to keep an eye for the CfP and submit your session's proposal then.

/cc @todaywasawesome @hblixt

**Reply by abdennour** [reply]
what does CfP meant ?And where to submit?

**Reply by jannfis** [reply]
Oh apologies, CfP == Call for Papers (or, Call for Participation). That's basically the phase when we know when and where ArgoCon '22 will be held, and we're reaching out to the community to submit proposals for interesting talks and sessions. By the time the CfP for ArgoCon '22 is issued, we will also let the community know how/where to submit their proposals.

It's too early for being more specific right now on those subjects.

**Reply by abdennour** [reply]
If you have slack or more interactive channel, so i can be notified ?

**Reply by todaywasawesome** [reply]
@abdennour Yes, you can join the Argo Slack. Instructions can be found here: https://argoproj.github.io/community/join-slack

If you register for ArgoCon '21 at the event page (even though it is passed) you will be notified of future events. https://www.techstrongevents.com/argocon21/1678035

**Comment by jannfis** [comment]
@abdennour FYI, https://events.linuxfoundation.org/argocon/

**Comment by abdennour** [comment]
Talk Proposal is Submitted !!! 🎉
@jannfis  looking forward for your support 🌹
